packages/pigeon/run_tests.sh

```diff
@@ -408,6 +403,3 @@ fi
 if [ "$should_run_android_unittests" = true ]; then
   run_android_unittests
 fi
-if [ "$should_run_formatter" = true ]; then
```

Comment:
The variable itself should be eliminated as well.

Reply:
done